README.Rmd

```diff
@@ -22,13 +22,156 @@ knitr::opts_chunk$set(
 
 ## Overview
```

Comment:
In general, you can consider writing the readme as I did the QC vignette; 
"protti" is always written in bold there and all the functions are written including brackets. So e.g. instead of `create_synthetic_data` I wrote `create_synthetic_data()`